### DISCUSSION TITLE
@dmikusa-pivotal Emeritus PHP Team Maintainer

### DISCUSSION THREAD

**Discussion by dmikusa** [paketo-buildpacks/feedback#136] [PHP Team]
Hi, I'm requesting to move me to the Emeritus PHP Team Maintainers group. It's been a blast working on the PHP buildpack and I'm so happy to see how far it's come. I am primarily working on other buildpacks now, so it seems like time to depart. Thanks

**Comment by sophiewigmore** [comment]
Thanks for all of your contributions and insights on the PHP buildpacks, Dan!

**Comment by dmikusa** [comment]
PR: https://github.com/paketo-buildpacks/community/pull/105. I've removed myself from the maintainer's team.
